programs/uxd/tests/unit_tests/utils/calculate_depositories_target_redeemable_amount.rs

```diff
@@ -0,0 +1,270 @@
+// Unit tests
+#[cfg(test)]
+mod test_calculate_depositories_target_redeemable_amount {
```

Comment:
not sure why this appears as an added new file, but this is just the renamed file below, with additional and polished prop test implementation